### PR TITLE
marp: 0.0.13 -> 0.0.14

### PR DESCRIPTION
**Pull request by puffnfresh**
I think 0.0.13 never worked. The 0.0.14 download is weird.

https://github.com/yhatt/marp/issues/258

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/office/marp/default.nix [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-      mkdir -p $out/lib/marp $out/bin
-      cp -r ./* $out/lib/marp
-      ln -s $out/lib/marp/Marp $out/bin
+    mkdir -p $out/lib/marp $out/bin
+    cp -r ./* $out/lib/marp
+    ln -s $out/lib/marp/Marp $out/bin
   '';
 
   postFixup = ''
+    chmod u+x $out/bin/Marp
```
Comment (c0bw3b) [inline pkgs/applications/office/marp/default.nix]: The default fixup phase should do that already, no?

Comment (Mic92) [reply pkgs/applications/office/marp/default.nix]: I don't think so. Sometimes there are some non-executable files in the same directory also this is dirty but some software just require that.

**Comment by Mic92** [comment]
Does this need a backport?

**Comment by puffnfresh** [comment]
@Mic92 yes, this is a security release

**Comment by aanderse** [comment]
Changelog: https://github.com/yhatt/marp/releases/tag/v0.0.14

**Comment by Mic92** [comment]
I get a different checksum:

```
fixed-output derivation produced path '/nix/store/iqmmn394wi5mmqyhp3b457pwrw9v7gif-0.0.14-Marp-linux-x64.tar.gz' with sha256 hash '0nklzxwdx5llzfwz1hl2jpp2kwz78w4y63h5l00fh6fv6zisw6j4' instead of the expected hash '0yx6qxrjfs4q4p6j11zxydr23r9y16l8arygr0yjl4dlzffdygxm'
```

**Comment by puffnfresh** [comment]
I reported a bug and they must have recreated the release :cry:

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build marp

**Pull request merged**